wgengine/netstack/netstack.go

```diff
@@ -843,6 +843,26 @@ func (ns *Impl) DialContextTCP(ctx context.Context, ipp netip.AddrPort) (*gonet.
 	return gonet.DialContextTCP(ctx, ns.ipstack, remoteAddress, ipType)
 }
 
+func (ns *Impl) DialContextTCPWithBind(ctx context.Context, localAddr, remoteAddr netip.AddrPort) (*gonet.TCPConn, error) {
```

Comment:
docs?

do you really need a `netip.AddrPort` (with dummy port) for the local addr? how about just `netip.Addr`?